ddR/R/zzz.R

```diff
@@ -16,5 +16,12 @@
 ###################################################################
 
 .onAttach <- function(libname, pkgname) {
-  packageStartupMessage("\nWelcome to 'ddR' (Distributed Data-structures in R)!\nFor more information, visit: https://github.com/vertica/ddR")
+    packageStartupMessage("\nWelcome to 'ddR' (Distributed Data-structures in R)!\nFor more information, visit: https://github.com/vertica/ddR")
+
+    # This should run if the default of parallel is being used as the
+    # driver.
+    # Maybe there's a better place to put this?
```

Comment:
1. Putting it here makes it more explicit that we're choosing parallel as the default.
2. `pdriver` is a model for other drivers, so it should avoid changing package level variables if possible.
3. There is some overhead to set up a parallel cluster, and this is a step in the direction of avoiding that overhead.